_source/_posts/2023-07-31-workflows-workshop.md

```diff
@@ -0,0 +1,939 @@
+---
+layout: blog_post
+title: "Enterprise Ready Workshop: Automate with with no-code Okta Workflows"
+author: max-katz
+by: advocate
+communities: [no-code,javascript]
+description: "Add automated reporting with no-code Okta Workflows. Enhance customer satisfaction by enabling custom automation."
+tags: [enterprise-ready-workshops, workflows, no-code, low-code]
+tweets:
+- ""
+image: blog/workflows-workshop/social.jpg
+type: awareness
+---
+
+
+{% include toc.md %}
```

Comment:
This should move to below the first paragraph (or in the intro)

Reply:
There's also A LOT of headers, so the TOC looks super overwhelming. We need to evaluate when headers are the correct choice, and when to use other forms of separation (such as bold titles for section) or ordered lists, etc.

Reply:
I moved the TOC to the end of the Introduction.